src/App.tsx

```diff
@@ -1,25 +1,141 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from "react";
+
+import {
+  Container,
+  CssBaseline,
+  Grid,
+  LinearProgress,
+} from "@material-ui/core";
+import Header from "./component/Header";
+
+//styles
+import "./App.css";
+
+import CoverPost from "./component/CoverPost";
+import MediaCard from "./component/CardComponent";
+import Footer from "./component/Footer";
+
+export type CardItemType = {
+  id: number;
+  category: string;
+  description: string;
+  image: string;
+  price: number;
+  title: string;
+};
+
+export type ShoppingCart = {
+  id: number;
+  itemQty: number;
+  itemName: string;
+  image: string;
+  price: number;
+};
 
 function App() {
+  const [sideDrawer, setsideDrawer] = useState(false);
```

Comment:
`setsideDrawer` should be proper camelcase `setSideDrawer`

Reply:
Implemented